Azure/azure-ad/ingest/parser.yml

```
@@ -0,0 +1,83 @@
+name: azure-ad
+pipeline:
+  - name: parsed_event
+    external:
+      name: json.parse-json
+  - name: azuread
+  - name: azuread-filters
+  - name: set_action_properties
+    filter: '{{ parsed_event.message.properties != null }}'
+  - name: set_if_message_properties
+    filter: '{{ parsed_event.message.properties != null }}'
+
+stages:
+  azuread:
+    actions:
+      - set:
+          service.type: "ldap"
+          service.name: "active directory"
+          log.hostname: "azureactivedirectory"
```

`host.hostname` is the standard field to hold the hostname
```suggestion
          log.hostname: "azureactivedirectory"
          host.hostname: "azureactivedirectory"
```